### DISCUSSION TITLE
Automation stop running instances

### DISCUSSION THREAD

**Discussion by Kri72** [home-assistant/frontend#18270] [Accessibility]
There is a service called automation.turn_off, that completely disable the automation. (I would have called it rather automation.disable)
Could we get a service that would just stop all running instances of a chosen automation without disabling the automation?
There are many cases when we might want to stop another running automation, based on another automation.

**Comment by frenck** [comment]
Hi there @Kri72 👋 

This is not a frontend change. If you want to suggest a feature like this, you should try our [Community Forum: Feature Requests](https://community.home-assistant.io/c/feature-requests).

If you have additional questions, feel free to join our [Discord chat server](https://discord.gg/c5DvZ4e).

Thanks! 👍
